packages/backend-core/src/index.ts

```diff
@@ -2,6 +2,7 @@ export * as configs from "./configs"
 export * as events from "./events"
 export * as migrations from "./migrations"
 export * as users from "./users"
+export * as usersUtils from "./users/utils"
```

Comment:
Nitpick, but could this be called `userUtils`?

Reply:
Changed 👍